### ISSUE TITLE
Is the project abandoned ?

### ISSUE THREAD

**Issue by luigi-derson** [storefront-foundation/react-storefront#198]
It looks like there is not activity since June / July. 
NextJS has released version 12 and it comes with a lot of improvements and features that can be implemented in the project.

I just want to know if it is worth it to contribute to the project or is it better to fork it and use my own version of it.

**Comment by slamer59** [comment]
A PR for it ? 
https://github.com/storefront-foundation/react-storefront/pull/199

**Comment by luigi-derson** [comment]
> A PR for it ? #199

Interesting, but we will see if they approve and merge it. Until then I'm using a forked version.

**Comment by geimsdin** [comment]
I guess it is abandoned, the advertised websites are not running reactstorefront anymore and no updates in almost one year... sad.

**Comment by adebiyi-al** [comment]
@luigi-derson @slamer59 @geimsdin , no it isn't abandoned yet, updates are currently been worked on.

**Comment by markbrocato** [comment]
Yep, definitely not abandoned, though it was dormant for some time. The team is finalizing an upgrade to the latest MUI and Next.JS, which should be out soon.

**Comment by migs35323** [comment]
soon(tm)
